support/allowed-prebuilts.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
There are still some roslyn prebuilts in 3.1.

Reply:
I disabled actually prebuilt enforcement with https://github.com/dotnet/source-build/pull/1383/commits/dc9a11ced5fd7fbf8bed5d8d9b460f0c708ed54c, I figured once we hit zero it should be easy to flip that back on and with your improvement we'll only need to run CI once to find all the extras.

Reply:
Got it.  Makes sense.